Hunting Queries/BehaviorAnalytics/Anomalous Password Reset.yaml

```diff
@@ -15,7 +15,7 @@ query: |
 
  BehaviorAnalytics
  | where ActionType == "Reset user password"
- | where ActivityInsights.FirstTimeUserPerformedAction == "True"
+ | where ActivityInsights contains "True"
```

Comment:
It is possible to use "has" here instead of "contains".  It is much more efficient.

Reply:
done